src/DotVVM.Framework/Resources/Scripts/postback/postback.ts

```diff
@@ -56,31 +50,53 @@ export async function postBack(
     } catch (err) {
 
         if (err instanceof DotvvmPostbackError) {
-            const r = err.reason;
-            const wasInterrupted = r.type == "handler" || r.type == "event";
-            const serverResponseObject =
-                r.type == "commit" && r.args ? r.args.serverResponseObject :
-                r.type == "network" ? r.err :
-                r.type == "serverError" ? r.responseObject :
-                null;
+            const wasInterrupted = isInterruptingErrorReason(err);
+            const serverResponseObject = extractServerResponseObject(err);
+            
+            if (wasInterrupted) {
+                // trigger postbackRejected event
+                const postbackRejectedEventArgs: DotvvmPostbackRejectedEventArgs = {
+                    ...options,
+                    serverResponseObject,
```

Comment:
this is redundant, when the postback is interrupted, there is no `serverResponseObject` and `response`